### DISCUSSION TITLE
How to locate and resolve the issue of automatically adding the .txt extension to downloaded files?

### DISCUSSION THREAD

**Discussion by GoodPhilipShi** [coder/code-server#6629] [Q&A]
I deployed a code-server service using Docker and accessed it through the Chrome browser. 
code-server: v4.8.3
Code: 1.72.1
When I download a file without an extension, such as filename1, it is successfully downloaded but the file name is changed to filename1.txt. 
Why is the .txt extension automatically added? 
Where can I find the download file logs and how can I locate and resolve this issue?"
Thanks！

**Comment by code-asher** [comment]
I am not sure, maybe something about the mime type?  It is probably an upstream issue.  Does this reproduce in Codespaces or vscode.dev?  If so I would recommend asking https://github.com/microsoft/vscode, they will know more.

**Reply by GoodPhilipShi** [reply]
Thank you very much for your reply.
Where can I see the relevant logs when downloading files from code server? It cannot be found in the developer tool of the browser or in the log of the code server container.

**Reply by code-asher** [reply]
I do not believe downloads are logged.  If you enable trace logging with `--log trace` then it will show you when files are read, which includes downloads, but it does not give you a way to distinguish opening a file versus downloading it.

Trace logging should show in code-server's stdout and in `~/.local/share/code-server/logs`.
